src/dmd/objc.d

```diff
@@ -131,6 +133,20 @@ Objc objc()
     return _objc;
 }
 
+
+/**
+ * This struct contains all data for a class declaration that is needed for the
+ * Objective-C integration.
+ */
+struct ObjcClassDeclaration
+{
+    /// `true` if this class is a metaclass.
+    bool isMeta = false;
+
+    /// The metaclass of this class.
+    ClassDeclaration metaclass;
```

Comment:
Why have `isMeta`? Isn't it locked to `metaclass !is null` ?

Reply:
I'll investigate if `isMeta` can be removed.

Reply:
@WalterBright as far as I can see, `isMeta` is necessary. I need to be differentiate on when a class is a metaclass and when a class is a regular class but where the metaclass hasn't been set yet.